applications/GeoMechanicsApplication/tests/test_element_lab/oedometer_ULFEM/README.md

```diff
@@ -0,0 +1,30 @@
+# Oedometer ULFEM test
+
+This test simulates an oedometer lab test. The UPwUpdatedLagrangianElement2D3N element is used to simulate the soil
+sample in a very simple 4-noded mesh, as displayed in the following figure:
+
+![MeshStructure](MeshStructure.svg)
+
+## Setup
+
+The test is performed in a single stage, with the following conditions:
+
+-   Constraints:
+    -   The X and Y displacement in the bottom two nodes (3 and 4) are fixed.
+    -   The X displacement in all nodes is fixed.
+-   Material:
+    -   The material is described using a linear elastic material with a GeoLinearElasticPlaneStrain2DLaw.
+-   Conditions:
+    -   A UPwFaceLoadCondition2D2N is added to the top of the cube (nodes 1 and 2).
+-   Loads:
+    -   A line load in the y direction is applied to the top of the cube (nodes 1 and 2), linearly ramping up from 0 to
+      -1000000 kPa in the time interval \[0, 1\].
+
+## Assertions
+
+In this test, there are two results checks:
+
+1. It is asserted that the effective stress is -1000000 kPa in the integration points of both elements in the y
+   direction and 0.0 in the other directions, due to the Poisson ration being 0.0.
+2. It is asserted that the displacement of one of the top nodes (1) is close to -0.0909090909516868 as a result of the
```

Comment:
I am not sure this is correct -0.0909090909516868 (I think it should be -0.1 = -1.e6/1.e7)

Reply:
We did a bit more digging here and it turns out it is due to the non-linear behavior of the geometry. Here we have a plot of displacement vs time, where the load ramps up from 0 to -100000000 in this time interval. Since we are using the updatedlagrange element, we see that the displacement is non-linear. 
![image](https://github.com/KratosMultiphysics/Kratos/assets/56549273/fa672436-3f59-424b-ba8f-748f6ac4f45a)


When setting the move_mesh flag to false (making the geometry more linear) we get the following graph:
![image](https://github.com/KratosMultiphysics/Kratos/assets/56549273/7160e172-9e63-4fac-8ddf-4198b9170afa)


The reason the diff order element approximately behaved as it should, was because the load was a lot smaller, meaning the displacement was still in the linear region.

For now we propose to keep the result like it is now. I added a line to explain this in the README.md to explain the number.

FYI @WPK4FEM